netci.groovy

```diff
@@ -2808,10 +2840,10 @@ def static shouldGenerateJob(def scenario, def isPR, def architecture, def confi
                 }
                 break
             case 'corefx_innerloop':
-                if (os != 'Windows_NT'|| architecture != 'x64') {
+                if ( (os != 'Windows_NT' && os != 'Ubuntu' &&  os != 'OSX10.12') ||  architecture != 'x64') {
                     return false
                 }
-                if(configuration != 'Release' && configuration != 'Checked') {
+                if(configuration != 'Checked') {
```

Comment:
I don't think there's any reason to restrict this to just Checked. Seems like we should just delete this condition and allow all of Debug/Checked/Released jobs.